.github/workflows/debianold.yml

```diff
@@ -0,0 +1,44 @@
+name: Debian OLD (Jessie) test
+
+on:
+  push:
+    branches:
+    - 'pre-release*'
+    - 'prerelease*'
+    - 'release-*'
+    - 'debian*'
+  pull_request:
```

Comment:
do we really want to run it on every pull_request? IMHO these kind of less common things should be run on a schedule (once a week? once a day? something like that..) or before releases.

Reply:
No, it's temporarily just to trigger the action. Will be removed once everything is settled.